plugins/catalog/src/components/createEditLink.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LocationSpec } from '@backstage/catalog-model';
+
+const determineBitBucketBranch = (url: string): string => {
+  const delimiter = '/';
+  const start = 6;
+  const result = url?.split(delimiter).slice(start)[0];
```

Comment:
well `.slice(x)[0]` sounds like `[x]` with extra steps :)

Reply:
Removed the whole thing in favor of git-url-parse but a good call out non the less.